ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -175,16 +175,25 @@
       </else-if>
       <!-- Pour les chapter paper-conference entry-dictionary entry-encyclopedia -->
       <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
-        <choose>
-          <if variable="container-author" match="none">
-            <text macro="editor" prefix=" dans "/>
-            <text variable="container-title" prefix=", " font-style="italic" text-case="capitalize-first"/>
-          </if>
-          <else-if variable="container-author" match="any">
-            <text macro="book_author" prefix=" dans "/>
-            <text variable="container-title" prefix=", " font-style="italic" text-case="capitalize-first"/>
-          </else-if>
-        </choose>
+        <group prefix="dans " delimiter=", ">
```

Comment:
Also, you can simplify

```xml
        <group prefix="dans " delimiter=", ">
          <choose>
            <if variable="container-author" match="none">
              <choose>
                <if variable="editor" match="any">
                  <text macro="editor"/>
                  <text variable="container-title" font-style="italic" text-case="capitalize-first"/>
                </if>
                <else>
                  <text variable="container-title" font-style="italic" text-case="capitalize-first"/>
                </else>
              </choose>
            </if>
            <else>
              <text macro="book_author"/>
              <text variable="container-title" font-style="italic" text-case="capitalize-first"/>
            </else>
          </choose>
        </group>
```

to

```xml
        <group prefix="dans " delimiter=", ">
          <choose>
            <if variable="container-author" match="none">
              <choose>
                <if variable="editor">
                  <text macro="editor"/>
                </if>
              </choose>
            </if>
            <else>
              <text macro="book_author"/>
            </else>
          </choose>
          <text variable="container-title" font-style="italic" text-case="capitalize-first"/>
        </group>
```

since you use `<text variable="container-title" font-style="italic" text-case="capitalize-first"/>` in all cases.